internal/scaffold/ansible/constants.go

```diff
@@ -28,4 +28,6 @@ const (
 )
 
 // Arrays can't be constants but this should be a constant
-var AnsibleDelims = [2]string{"[[", "]]"}
+var (
+	AnsibleDelims = [2]string{"[[", "]]"}
+)
```

Comment:
:+1:

Reply:
This isn't the correct fix for the lint error, which was
```
internal/scaffold/ansible/constants.go:30:1: comment on exported var `AnsibleDelims` should be of the form `AnsibleDelims ...` (golint)
```

The correct fix is:
```go
// AnsibleDelims is a slice of two strings representing the left and right delimiters for ansible templates. Arrays can't be constants but this should be a constant.
var AnsibleDelims = [2]string{"[[", "]]"}
```